amundsen_application/static/js/components/NavBar/index.tsx

```diff
@@ -50,10 +50,12 @@ export class NavBar extends React.Component<NavBarProps> {
         <div className="row">
           <div className="nav-bar">
             <div id="nav-bar-left" className="nav-bar-left">
-              {
-                AppConfig.logoPath &&
-                <img id="logo-icon" className="logo-icon" src={AppConfig.logoPath} />
-              }
+              <Link to={`/`}>
+                {
+                  AppConfig.logoPath &&
+                  <img id="logo-icon" className="logo-icon" src={AppConfig.logoPath} />
+                }
```

Comment:
Instead of adding a second link it would be cleaner to put the logo inside the original link and verify that it looks okay with/without the logo.

Reply:
👍 will do